apps/src/code-studio/components/progress/UnitOverview.jsx

```diff
@@ -150,10 +153,15 @@ class UnitOverview extends React.Component {
       !!scriptId &&
       isScriptHiddenForSection(hiddenLessonState, selectedSectionId, scriptId);
 
+    const showUnversionedRedirectWarningDialog =
+      showUnversionedRedirectWarning && !this.state.showRedirectDialog;
```

Comment:
Could you add comments here or in UnversionedScriptRedirectDialog to help distinguish this from legit redirects from urls like /s/coursea or /courses/csp ?